tests/test_models.py

```diff
@@ -2348,25 +2348,6 @@ def test_get_assignments_many_assignments(self):
         actual_assignments = list(SystemWideEnterpriseUserRoleAssignment.get_assignments(test_user))
         self.assertEqual(expected_assignments, actual_assignments)
 
-    def test_get_assignments_no_context_or_link(self):
-        """
-        Test the scenario where a user has an assignment record,
-        but the record has no enterprise_customer_id, applies_to_all_contexts is False,
-        and the user is not linked to any enterprise.
-        """
-        user_with_no_link = factories.UserFactory(email='zelda@example.com')
-        SystemWideEnterpriseUserRoleAssignment.objects.get_or_create(
-            user=user_with_no_link,
-            role=roles_api.learner_role(),
-            enterprise_customer=None,
-            applies_to_all_contexts=False,
-        )
-
-        self.assertEqual(
```

Comment:
making a note here that i did examine https://github.com/openedx/edx-enterprise/blob/master/enterprise/management/commands/update_role_assignments_with_customers.py and it appears that we no longer create role assignments without either an enterprise_customer or applies_to_all_contexts; therefore this test should be safe to remove (without losing functional coverage)